packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added `config import` command that imports team config files from a local path or web URL. [#1083](https://github.com/zowe/zowe-cli/issues/1083)
+
```

Comment:
Should we add the Help Examples added to `zowe config` ?
Looking at the second bullet on https://github.com/zowe/imperative/blob/next/CHANGELOG.md#500-next202109281439